homeassistant/components/stream/const.py

```diff
@@ -15,8 +15,9 @@
 OUTPUT_IDLE_TIMEOUT = 300  # Idle timeout due to inactivity
 
 NUM_PLAYLIST_SEGMENTS = 3  # Number of segments to use in HLS playlist
-MAX_SEGMENTS = 4  # Max number of segments to keep around
+MAX_SEGMENTS = 5  # Max number of segments to keep around
 MIN_SEGMENT_DURATION = 1.5  # Each segment is at least this many seconds
+MIN_SEGMENT_DURATION_LL_HLS = 6  # With ll-hls we can use longer segments
```

Comment:
When we use LL-HLS we no longer have to use really short segments as the latency should be based on the part duration as opposed to the segment duration. Longer segments can reduce the amount of processing and messaging overhead from using more segments (at the expense of using more memory per segment).